src/os_posix/os_alloc.c

```diff
@@ -9,6 +9,27 @@
 #include "wt_internal.h"
 
 /*
+ * On systems with poor default allocators for allocations greater than 16 KB,
+ * we provide an option to use TCMalloc explicitly.
+ * This is important on Windows which does not have a builtin mechanism
+ * to replace C run-time memory management functions with alternatives.
+ */
+#ifdef HAVE_LIBTCMALLOC
+#include <gperftools/tcmalloc.h>
```

Comment:
@markbenvenuto, OK, there's definitely a mismatch here -- we're including `gperftools/tcmalloc.h`, and we tested for `google/tcmalloc.h`, which is correct?

Looking around the net makes me think it's `gperftools`, so I'll go with that for now, please fix it if I'm wrong.